plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlConnectorTest.java

```diff
@@ -78,7 +175,7 @@ public void testViews()
     }
 
     @Test
-    public void testInsert()
+    public void testSimpleInsert()
```

Comment:
Is this a duplicate of `testInsert` and can be removed?

Reply:
Looks like the insert tests is a dup, I'll remove it. There's also a `testViews` which makes me think that views are supported, so I tried enabling those tests too.